packages/scaffolding-cli/templates/src/ssr/pages/_app.tsx

```diff
@@ -60,25 +62,22 @@ class _App extends App<AppWithStore> {
                     <title>Amido Stacks SSR</title>
                 </Head>
 
-                <Provider store={store}>
-                    <ThemeProvider theme={theme}>
-                        {/* CssBaseline kickstart an elegant, consistent, and simple baseline to build upon. */}
-                        <CssBaseline />
-                        <Component {...props} />
-                        <Notifier />
-                    </ThemeProvider>
-                </Provider>
+                <ThemeProvider theme={theme}>
+                    {/* CssBaseline kickstart an elegant, consistent, and simple baseline to build upon. */}
+                    <CssBaseline />
+                    <Component {...props} />
+                    <Notifier />
+                </ThemeProvider>
             </>
         )
     }
 }
 
 const appInsightsConfig = {
-    instrumentationKey: getConfig().serverRuntimeConfig
-        .APPINSIGHTS_INSTRUMENTATIONKEY,
+    instrumentationKey: process.env.APPINSIGHTS_INSTRUMENTATIONKEY,
```

Comment:
Why are we pulling straight from environment variable?

Reply:
Yes